frontend/src/pages/homepage.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Navbar from "../utils/navbar";
+import { PageTitle } from "../utils/common";
+
+export const Homepage = () => {
+    return (
+        <>
+            <Navbar />
```

Comment:
I can smell that `Navbar` will be boilerplated everywhere, except in some specific pages.
Can we make an intermediate component (something like AuthenticatedUserContainer) for it?

Reply:
Incoming in the next PR

Reply:
what a cliffhanger